src/main/java/org/broadinstitute/hellbender/utils/Utils.java

```diff
@@ -1137,4 +1139,12 @@ public static void forceJVMLocaleToUSEnglish() {
         final List<T> sorted = values.stream().sorted().collect(Collectors.toList());
         return sorted.get(sorted.size() / 2);
     }
+
```

Comment:
Can you refactor into a class that implements `Iterable`, so that this can be used with for-each loops?

Reply:
Also should implement `Closeable` so that we can ensure the stream gets closed.

Reply:
Refactored. It now implements `Iterable` and `AutoCloseable` (for new code it's preferable to implement that instead of `Closeable`).